run_dir/design/worksets.html

```diff
@@ -54,16 +54,16 @@ <h2 id="{{ worksets }}" class="worksets_page_heading"> <small>Worksets from the
                          </td>
                      {% elif onekey[1] == 'workset_name'%}
                          <td>
-                         <a href="/workset/{{ onews }}" style="color:#000000;">{{ onews }}</a>
+                         <a href="/workset/{{ onews }}" style="color:#000000;" class="text-decoration-none">{{ onews }}</a>
                          </td>
                      {% elif onekey[1] == 'projects'%}
                          <td>
                          {% for onepj in ws_data.get(onews).get(onekey[1]) %}
                             {% if ws_data.get(onews).get(onekey[1]).get(onepj).get('status') == 'Open' %}
-                                <a href="/project/{{ onepj }}" class="text-success" data-toggle="tooltip" title="Status: Open"><i class="glyphicon glyphicon-folder-open">&nbsp;</i>{{ ", ".join([ws_data.get(onews, {}).get(onekey[1], {}).get(onepj, {}).get('project_name', 'NA'), onepj]) }}</a>
+                                <a href="/project/{{ onepj }}" class="text-success text-decoration-none" data-toggle="tooltip" title="Status: Open"><i class="fa fa-folder-open">&nbsp;</i>{{ ", ".join([ws_data.get(onews, {}).get(onekey[1], {}).get(onepj, {}).get('project_name', 'NA'), onepj]) }}</a>
                             {% end %}
                             {% if ws_data.get(onews).get(onekey[1]).get(onepj).get('status') == 'Closed' %}
-                                <a href="/project/{{ onepj }}" class="text-warning" data-toggle="tooltip" title="Status: Closed"><i class="glyphicon glyphicon-folder-close">&nbsp;</i>{{ ", ".join([ws_data.get(onews, {}).get(onekey[1], {}).get(onepj, {}).get('project_name', 'NA'), onepj]) }}</a>
+                                <a href="/project/{{ onepj }}" class="text-warning text-decoration-none" data-toggle="tooltip" title="Status: Closed"><i class="fa fa-folder">&nbsp;</i>{{ ", ".join([ws_data.get(onews, {}).get(onekey[1], {}).get(onepj, {}).get('project_name', 'NA'), onepj]) }}</a>
```

Comment:
Could we maybe change the color of the closed worksets? I know it's the default text-warning class, but it's a bit bright, no?

Reply:
Yeah I think I agree with you. We could either just change it specificly for this one or we could modify the warning color in the bootstrap file we use. Do you think the warning color will always be too bright?

Reply:
Yes, as a bg-color it would look nice, but not for plain text with a white bg...

Reply:
All the background colours are brighter in the new bootstrap. Does this look better to you?
<img width="641" alt="Screen Shot 2020-12-18 at 10 49 43" src="https://user-images.githubusercontent.com/11080857/102600245-e1ea8700-411e-11eb-88ce-80d9438bc8d8.png">

Reply:
Yes, that looks better I think!

Reply:
It up on stage now.